features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -7,114 +8,109 @@
 using BugsnagUnity;
 using BugsnagUnity.Payload;
 using UnityEngine.SceneManagement;
-using UnityEngine.UI;
 using System.IO;
 
 #if UNITY_EDITOR
 using UnityEditor;
 #endif
 
+[Serializable]
+public class Command
+{
+    public string action;
+    public string scenarioName;
+}
+
 public class Main : MonoBehaviour
 {
 
 
 #if UNITY_STANDALONE_OSX
 
+    [DllImport("NativeCrashy")]
+    private static extern void PreventCrashPopups();
+
     [DllImport("NativeCrashy")]
     private static extern void crashy_signal_runner(float num);
 
 #endif
 
+    private const string API_KEY = "a35a2a72bd230ac0aa0f52715bbdc6aa";
     private Dictionary<string, string> _webGlArguments;
 
 
     private string _fakeTrace = "Main.CUSTOM () (at Assets/Scripts/Main.cs:123)\nMain.CUSTOM () (at Assets/Scripts/Main.cs:123)";
 
-    private float _closeTime = 5;
+#if UNITY_STANDALONE || UNITY_WEBGL
+    private string _mazeHost = "http://localhost:9339";
+#else
+    private string _mazeHost = "http://bs-local.com:9339";
+#endif
 
-    public void Start()
+    IEnumerator RunNextMazeCommand()
```

Comment:
Nit, would be nice if Main.Start was the first method in the file to save future devs time working out entry points etc...

Reply:
Moved.